index.html

```diff
@@ -0,0 +1,87 @@
+html> 
+   <head> 
+      <script> 
+         
```

Comment:
Don't forget we learned how to make script.js file for the javascript and import into your html